packages/dashmate/templates/platform/gateway/envoy.yaml.dot

```diff
@@ -0,0 +1,486 @@
+!ignore filters: &filters
+  - name: envoy.http_connection_manager
```

Comment:
Shall we rate limit incoming connections by IP? RIght now we limit only incoming requests by IP.

Reply:
We agreed that this must be done on host level or with external services and we can't do anything with this at dashmate level.